### PR TITLE
Implement navigation from stub to the schema

### LINKED ISSUE #193: Navigation from stub to the schema
Currently it is possible to go from gRPC method/service implementation to the corresponding schema definition, and from the schema definition to the stub (client) calls.
But it is impossible to navigate from client stub to the schema, which is very useful to find the server-side implementation for given client-side gRPC invocation

### PR DESCRIPTION
**Pull request by fan-tom**
Closes #193 
This PR adds line markers, similar to the implementation of gRPC service and method, to the generated stubs (of all kinds), see attached screenshots
изображение
изображение
изображение
изображение

In addition, it adds gRPC method calls, made using coroutine-based stubs, to the usages of the corresponding gRPC methods, so, it is now possible to navigate from gRPC schema to the client-side gRPC call made using coroutine-based stub.

### CODE REVIEW

File: src/main/kotlin/io/kanro/idea/plugin/protobuf/java/JavaLineMarkerProvider.kt [hunk 1]
```diff
@@ -29,15 +29,19 @@ class JavaLineMarkerProvider : RelatedItemLineMarkerProvider() {
...
         when (val parent = identifier.uastParent) {
             is UClass -> {
-                val service = findServiceProtobufDefinition(parent) ?: return
+                val service = findServiceProtobufDefinition(parent)
+                    ?: findServiceProtobufDefinitionForStub(parent)
+                    ?: return
                 val builder: NavigationGutterIconBuilder<PsiElement> =
                     NavigationGutterIconBuilder.create(ProtobufIcons.IMPLEMENTING_SERVICE)
                         .setTargets(service)
                         .setTooltipText("Implementing")
```
Comment (fan-tom) [inline src/main/kotlin/io/kanro/idea/plugin/protobuf/java/JavaLineMarkerProvider.kt:38]: I'm not sure whether it is ok to use word "Implementing" (and the corresponding icons) for stubs, @devkanro please advise here

Comment (devkanro) [reply src/main/kotlin/io/kanro/idea/plugin/protobuf/java/JavaLineMarkerProvider.kt:38]: Let's keep it now, I'll make their own icons for stubs if I have time

**Comment by fan-tom** [comment]
Hey @devkanro, maybe release this PR?

**Comment by devkanro** [comment]
I will release it in next week with some bug fix

**Pull request merged**